lib/MailQueueHandler.php

```diff
@@ -405,6 +407,12 @@ protected function sendEmailToUser($userName, $email, $lang, $timezone, $maxTime
 		try {
 			$this->mailer->send($message);
 		} catch (\Exception $e) {
+			$this->logger->logException($e, [
+							'message' => 'Failed sending email to user "{user}"',
+							'user' => $user,
```

Comment:
Log the exception with `$this->logger->logException($e);` and log a additional warning `$this->logger->warning('Failed to send notification email for user {user}', ['user' => $userName]);`. You cannot use `$user` because it's a IUser object.